client/src/views/v2/CameraSettingsView.js

```diff
@@ -10,6 +11,9 @@ import CameraSettings from 'components/v2/CameraSettings';
 export default function CameraSettingsView() {
   return (
     <ContentPage title="Camera Settings">
+      <div className="mb-4">
+        <CameraRecording devices="primary,secondary" />
```

Comment:
You wouldn't have that nasty string split inside `CameraRecording` if you had just passed this in as an array.

`        <CameraRecording devices=["primary", "secondary"] />`

Reply:
Done
https://github.com/monash-human-power/dashboard/pull/74/commits/7c9a27d57887c765470f6eaf8879817115defbc9

I should have known that React would have a way to pass objects as props instead of just a string haha.